static/css/drawer.css

```diff
@@ -164,11 +165,21 @@ h1, legend,
     margin-top: 0;
     margin-left: 42px;
     padding-top: 10px;
-    padding-right: 150px;
+    padding-right: 90px;
     white-space: nowrap;
     text-overflow: ellipsis;
     overflow: hidden;
 }
+#iob-cob h1 {
```

Comment:
this is targeting `dev` there is no `#iob-cob` yet, no reason to the adjusting it's style here